docs/source/markdown/options/read-only-tmpfs.md

```diff
@@ -4,4 +4,20 @@
 ####> are applicable to all of those.
 #### **--read-only-tmpfs**
 
-If container is running in **--read-only** mode, then mount a read-write tmpfs on _/dev_, _/dev/shm_, _/run_, _/tmp_, and _/var/tmp_. The default is **true**.
+When running --read-only containers mount a read-write tmpfs on _/dev_, _/dev/shm_, _/run_, _/tmp_, and _/var/tmp_. The default is **true**.
+
+| --read-only | --read-only-tmpfs |  /   | /run, /tmp, /var/tmp|
+| ----------- | ----------------- | ---- | ----------------------------------- |
+| true        |  true             | r/o  | r/w                                 |
+| true        |  false            | r/o  | r/o                                 |
+| false       |  false            | r/w  | r/w                                 |
+| false       |  true             | r/w  | r/w                                 |
+
+When --read-only=True and --read-only-tmpfs=True additional tmpfs are mounted on
+the /tmp, /run, and /var/tmp directories.
+
+When --read-only=True and --read-only-tmpfs=False /dev and /dev/shm are marked
+Read/Only and no tmpfs are mounted on /tmp, /run and /var/tmp. The directories
+are exposed from the underlying image, meaning they are read-only by default.
+
+In all other cases where --read-only=false the /dev and /dev/shm are read/write and the /tmp, /run, and /var/tmp are not mounted over by default.
```

Comment:
It seems to be common style to put command and flags into bold blocks in all man pages, i.e. `**--read-only**`
I also find it weird to use the upper case True/False, I think we generally stick to the lowercase versions everywhere. 

And lastly the last sentence doesn't look right grammatically to me. I am not a native speaker of course but without extra context I don't think I am able to understand this.